presto-oracle/pom.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.prestosql</groupId>
+        <artifactId>presto-root</artifactId>
+        <version>325-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>presto-oracle</artifactId>
+    <description>Presto - Oracle Connector</description>
+    <packaging>presto-plugin</packaging>
+
+    <properties>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+        <oracle.container.version>1.12.3</oracle.container.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.oracle.ojdbc</groupId>
+            <artifactId>ojdbc8</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.testcontainers</groupId>
```

Comment:
This is cool, although we already have something like this.

We use `io.prestosql.testing.docker.DockerContainer`.
See `TestingPostgreSqlServer` for usage example.

Reply:
done

Reply:
May I try to convince you that Testcontainers is a better long-term choice? 😊

We (Testcontainers, I am a co-maintainer) have a large community (with users including but not limited to Spring, Apache Projects like Pulsar/BookKeeper, Camel, eBay Marketing, IBM OSS, Red Hat, Spotify, Micronaut, Zipkin, to name a few) of people focused on using Docker for testing. The project is 100% OSS, not backed by any (opinionated) company or anything.

The project has a long history and we collected as many pitfalls and knowledge as (I think) possible. Some examples:
- We support running on Linux, Mac (docker-machine and Docker for Mac), Windows (docker-machine and Docker for Windows with npipe), docker-in-docker combinations and various CI servers. For that, we have a set of (pluggable) environment discovery strategies
-Everything we start is removed after the test. For that, we use both, shutdown hook **and** a sidecar Golang container that monitors the connection to the tests' host and if the tests JVM is gone (e.g. `kill -9 $MAVEN_PID`), it will ensure that the containers are destroyed
- Some of our modules (Couchbase, DB2, Kafka, and more!) are a result of collaboration with the original authors of these technologies who helped us to make sure that they work correctly (and fast)
- There are some advanced features like container-to-host communication, or the reusable containers (in preview)

It is very cool to see `io.prestosql.testing.docker.DockerContainer`! I believe if you agree to use Testcontainers, we may learn from your experience too and the whole community will benefit from it 👍 

P.S. I could not not notice that you use Spotify's docker client, which is no longer actively developed. Testcontainers used to use it, but then we migrated to docker-java (and later joined as maintainers of docker-java to ensure that it works well for our users)

Thanks! :)

Reply:
@bsideup We tend to introduce one change at a time. Here pull request is about the Oracle connector. 

I fine with exploring new ways (possibly better) for stuff that is already in place. Would you like to open a pull request where you could replace usage `io.prestosql.testing.docker.DockerContainer` with Testcontainers? So we could discuss usage of Testcontainers there, out of oracle connector.

Reply:
@bsideup thanks for nice summary. Yes, keep this independent from oracle PR.
I suggest that before investing in a PR, let's talk on `#dev` on our slack (https://prestosql.io/slack.html) first.  Looking forward to talk to you there!